backend/plugins/payment_api/src/apis/khanbank/api.ts

```
@@ -0,0 +1,132 @@
+  }
+
+  async cancelInvoice(invoice: ITransactionDocument) {
+    try {
+      //   await this.request({
+      //     method: 'DELETE',
+      //     path: `${PAYMENTS.qpay.actions.invoice}/${invoice.response.invoice_id}`,
+      //     headers: await this.getHeaders(),
+      //   });
+    } catch (e) {
+      return { error: e.message };
+    }
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)